src/actions/ScanActions.js

```diff
@@ -50,24 +50,22 @@ import { launchDeepLink } from './DeepLinkingActions.js'
 export const doRequestAddress = async (dispatch: Dispatch, currencyWallets: { [walletId: string]: EdgeCurrencyWallet }, link: RequestAddressLink) => {
   dispatch({ type: 'DISABLE_SCAN' })
   const { assets, post, redir, payer } = link
-
   try {
     // Check if all required fields are provided in the request
-    if (assets.length === 0) throw new Error('No currencies found in request for payment address')
-    if (post == null && redir == null) throw new Error('Post or redir address not found in request for payment address')
-    if (post != null && redir != null) throw new Error('Both post and redir address were specified in request for payment address')
+    if (assets.length === 0) throw new Error(s.strings.bitwage_error_no_currencies_found)
```

Comment:
Remove reference to `bitwage` this is NOT a bitwage specific feature. It's the Request for Address protocol